src/manage_nsfs/nc_lifecycle.js

```diff
@@ -3,24 +3,63 @@
 
 const dbg = require('../util/debug_module')(__filename);
 const _ = require('lodash');
+const path = require('path');
 const util = require('util');
 const P = require('../util/promise');
 const config = require('../../config');
 const nb_native = require('../util/nb_native');
 const NsfsObjectSDK = require('../sdk/nsfs_object_sdk');
+const native_fs_utils = require('../util/native_fs_utils');
 const ManageCLIError = require('./manage_nsfs_cli_errors').ManageCLIError;
-const path = require('path');
 const { throw_cli_error, get_service_status, NOOBAA_SERVICE_NAME } = require('./manage_nsfs_cli_utils');
+const { is_desired_time, record_current_time } = require('./manage_nsfs_cli_utils');
 
-// TODO: 
+// TODO:
 // implement 
 // 1. notifications
 // 2. POSIX scanning and filtering per rule
 // 3. GPFS ILM policy and apply for scanning and filtering optimization
 
+const CLUSTER_LOCK = 'cluster.lock';
```

Comment:
Can we add a check to make sure that `config.LIFECYCLE_LOGS_DIR` is never the same as `GLACIER_LOGS_DIR`. Because if someone somehow makes them same then all the glacier operations and lifecycle operations will start to run sequentially because they will have the same path.

This might be ignored as well and we can just say that the admin configured the system inccorrectly.